src/parser/mod.rs

```diff
@@ -5867,10 +5888,10 @@ impl<'a> Parser<'a> {
 
     /// Parse a possibly qualified, possibly quoted identifier, e.g.
     /// `foo` or `myschema."table"
-    pub fn parse_object_name(&mut self) -> Result<ObjectName, ParserError> {
+    pub fn parse_object_name(&mut self, in_table_clause: bool) -> Result<ObjectName, ParserError> {
```

Comment:
Can you please add documentation to this function explaining the new paramter and some context of what it is used for?